src/boot/stylesheets/main.scss

```diff
@@ -690,12 +690,41 @@ body {
     				display: block;
     			}
     		}
+
+			code {
+				font: $code;
+				font-size: 90%;
+				color: $code-color;
```

Comment:
Nice, we don't really have a style for `code` in Calypso. Looks good, but with some color changes. First, I wouldn't use variables since this is a one-off deal and we try to keep the colors.scss file pretty clean.

Change the colors to this:

```
color: darken( $gray, 20% );
background: $gray-light;
border: 1px solid lighten( $gray, 20% );
```